Neos.Flow/Classes/Core/Booting/Scripts.php

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
PhpStorm likes this additional space by default, but we don't (according to our CGL) You can change this in PhpStorm in the Code Style Settings